includes/wikia/models/NavigationModel.class.php

```diff
@@ -1,14 +1,15 @@
 <?php
+
 /**
  * Model for Wiki Navigation
  *
  * @author Jakub "Student" Olek
  */
-
 class NavigationModel extends WikiaModel {
 	const WIKIA_GLOBAL_VARIABLE = 'wgOasisGlobalNavigation';
 	const WIKI_LOCAL_MESSAGE = 'Wiki-navigation';
-	const CACHE_TTL = 10800; // 3 hours
+	// 3 hours
+	const CACHE_TTL = 10800;
 
 	const version = '1';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this constant "version" to match the regular expression ^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS115)